podpac/core/coordinates/test/test_coordinates.py

```diff
@@ -2094,3 +2095,66 @@ def test_transform_same_crs_same_result(self):
 
         assert_array_equal(c2["lat"].coordinates, c1["lat"].coordinates)
         assert_array_equal(c2["lon"].coordinates, c1["lon"].coordinates)
+
+
+class TestCoordinatesMethodSimplify(object):
+    def test_simplify_array_to_uniform(self):
+        c1 = Coordinates([[1, 2, 3, 4], [4, 6, 8]], dims=["lat", "lon"])
+        c2 = Coordinates([[1, 2, 3, 5], [4, 6, 8]], dims=["lat", "lon"])
+        c3 = Coordinates([clinspace(1, 4, 4), clinspace(4, 8, 3)], dims=["lat", "lon"])
+
+        # array -> uniform
+        assert c1.simplify() == c3.simplify()
+
+        # array -> array
+        assert c2.simplify() == c2.simplify()
+
+        # uniform -> uniform
+        assert c3.simplify() == c3.simplify()
+
+    @pytest.mark.skip(reason="not implemented, spec uncertain")
+    def test_simplify_stacked_to_unstacked_arrays(self):
+        stacked = Coordinates([np.meshgrid([1, 2, 3, 5], [4, 6, 8])], dims=["lat_lon"])
+        unstacked = Coordinates([[1, 2, 3, 5], [4, 6, 8]], dims=["lat", "lon"])
+
+        assert stacked.simplify() == unstacked
+        assert unstacked.simplify() == unstacked
+
+    def test_stacked_to_unstacked_uniform(self):
+        stacked = Coordinates([np.meshgrid([4, 6, 8], [1, 2, 3, 4])[::-1]], dims=["lat_lon"])
+        unstacked_uniform = Coordinates([clinspace(1, 4, 4), clinspace(4, 8, 3)], dims=["lat", "lon"])
+
+        # stacked grid -> uniform
+        assert stacked.simplify() == unstacked_uniform
+
+        # uniform -> uniform
+        assert unstacked_uniform.simplify() == unstacked_uniform
+
+    def test_stacked_to_affine(self):
+        geotransform_rotated = (10.0, 1.879, -1.026, 20.0, 0.684, 2.819)
+        affine = Coordinates([AffineCoordinates(geotransform=geotransform_rotated, shape=(4, 6))])
+        stacked = Coordinates([StackedCoordinates([affine["lat_lon"]["lat"], affine["lat_lon"]["lon"]])])
+
+        # stacked -> affine
+        assert stacked.simplify() == affine
```

Comment:
@mpu-creare Here is the test that fails due to the geotransform being not quite close enough after the roundtrip.